### PR TITLE
Added support for gzip from telegraf

### PR DESCRIPTION
**Pull request by megalan247**
Hey everyone, telegraf has support for posting gzipped data to influx, to limit data transfer. However this exporter doesn't have support for decompressing gzipped posted data.

I added this here.

### CODE REVIEW

**Comment by matthiasr** [comment]
This looks good in general. Can you please fix the style checks by running `go fmt main.go`? Also add the DCO sign-off as detailed [here](https://github.com/prometheus/influxdb_exporter/pull/78/checks?check_run_id=1556959842).

Finally, I would really like to have a test that exercises this (ideally using an official InfluxDB client so we know that it works with that), could you add that?

**Comment by megalan247** [comment]
Hey, sorry for the delay. I fixed the style checks and added the DCO signoff, but unfortunately I am quite new to go and dont have the time to create the test you mentioned :(

**Comment by matthiasr** [comment]
No worries, I added a follow-up issue for anyone who wants to get their feet wet.

File: main.go [hunk 1]
```diff
@@ -14,8 +14,10 @@
 package main
 
 import (
+	"compress/gzip"
 	"encoding/json"
 	"fmt"
+	"gopkg.in/alecthomas/kingpin.v2"
```
Comment (matthiasr) [inline main.go:20]: cosmetic: this shouldn't be in the middle of all the standard library imports, I'll fix that up after merge.

**Pull request merged**